src/constructs/lambda/lambda.ts

```diff
@@ -63,6 +66,13 @@ export class GuLambdaFunction extends Function {
       });
     });
 
+    if (props.errorPercentageMonitoring) {
+      new GuLambdaErrorPercentageAlarm(scope, "error-percentage-alarm-for-lambda", {
```

Comment:
I think we've used `TitleCase` for ids so far. Any objection to continue with that pattern here? And/or should we have a linter rule for future?

Reply:
No objections, just an apology that I also broke this convention in my previous PR! 🙈 

If we want consistency here then we probably do want a linter rule (or a type, if that's possible?!) to enforce this convention.

Reply:
Although thinking about this some more, teams will need to be able to bring their existing ids (regardless of casing) in order to preserve resources that have been created by CFN, so maybe we can't enforce this too strictly?

Reply:
IIUC linting will only be scoped to this repository - consumers of this library would need to perform linting for themselves, bringing their own rules. That is, I think linting is still good here.

Reply:
Let's add a linter rule in a different PR to keep things simple.